tests/tests_common.py

```diff
@@ -521,8 +521,8 @@ def test_array_size_one_value(self):
         self.assertEqual([1], val)
 
     def test_use_namespace(self):
-        idx = self.opc.get_namespace_index("urn:freeopcua:python:server")
-        self.assertEqual(idx, 1)
+        idx = self.opc.get_namespace_index("http://opcfoundation.org/UA/")
+        self.assertEqual(idx, 0)
```

Comment:
OK that test was also outdated. usually users should only create nodes in custom namespace. so the best is probably to  set again the application URI here 
```
self.opc.set_application_uri("urn:freeopcua:python:server")
 idx = self.opc.get_namespace_index("urn:freeopcua:python:server")
 self.assertEqual(idx, 1)
```
that should work

Reply:
otherwise everything is fine

Reply:
@oroulet The problem with this solution is with the Client-side test of this part, because then `opc` is a Client instance which has no `set_application_uri()` member. Tests say the same.

Reply:
ok then what about:
```
idx_orig = 1
uri = self.opc.get_namespace_array()[idx]
idx = self.opc.get_namespace_index(uri)
self.assertEqual(idx_orig, idx)
```
that way we test several methods

Reply:
sounds good :+1: